Formula/redis.rb

```diff
@@ -1,6 +1,6 @@
 class Redis < Formula
   desc "Persistent key-value database, with built-in net interface"
-  homepage "http://redis.io/"
+  homepage "https://redis.io/"
   url "http://download.redis.io/releases/redis-3.2.8.tar.gz"
   sha256 "61b373c23d18e6cc752a69d5ab7f676c6216dc2853e46750a8c4ed791d68482c"
   head "https://github.com/antirez/redis.git", :branch => "unstable"
```

Comment:
I believe the branch should be [3.2](https://github.com/antirez/redis/tree/3.2) ?